pkg/common/client.go

```diff
@@ -799,6 +799,25 @@ func (c *Client) Ping() error {
 	return nil
 }
 
+// Get the service account name from a client
+func serviceAccountName(clientName string) string {
+	return "service-account-" + clientName
```

Comment:
This should not be hardcoded. Please try to get the username from [REST API](https://www.keycloak.org/docs-api/15.0/rest-api/index.html#_getserviceaccountuser).